content/docs/04.5-devx/05-app-profile.md

```diff
@@ -93,4 +96,21 @@ spec:
 
 </WarningBox>
 
+# App Profile Macros
+
+Pack macros are the variables defined in the App Profile, and these variables are resolved only at the cluster deployment time.
+<br />
+
+```
+{{.spectro.system.[VARIABLE_NAME]}}
+```
+**Supported Variables**
+* appprofile.name
+* appprofile.uid
+* appdeployment.name
+* appdeployment.uid
+* appdeployment.tiername : Resolves to a string value in the format `<deployment name>-<tier name>`
+
```

Comment:
We might want to consider a table here. If the last variable has a definition, then the rest should also have a definition. That's why a table is helpful in this context, as it will be easier for people to see the variables and their definitions.